### DISCUSSION TITLE
Which regions can I deploy Supabase in?

### DISCUSSION THREAD

**Discussion by awalias** [supabase/supabase#4815] [Questions] [answered]
On the hosted platform, which are the regions (datacenters) that I can deploy in?

**Comment by awalias** [comment, marked as answer]
The most up to date list of regions is in the New Project form inside the [dashboard](https://app.supabase.com), but here is the list as of today (2022-01-05):

AWS - West US (North California)
AWS - East US (North Virginia)
AWS - Canada (Central)
AWS - West EU (Ireland)
AWS - West EU (London)
AWS - Central EU (Frankfurt)
AWS - South Asia (Mumbai)
AWS - Southeast Asia (Singapore)
AWS - Northeast Asia (Tokyo)
AWS - Northeast Asia (Seoul)
AWS - Oceania (Sydney)
AWS - South America (Sao Paulo)

Currently, the database and your APIs are all deployed inside the same datacenter and in the same availability zone. Please contact us via the [help form](https://app.supabase.io/support/new) in the dashboard for any clarifications or additional requirements.

**Reply by jinsley8** [reply]
Anyone know approximately where the Canada datacenter is located?

Trying to decide whether that or East US is closest to the rest of my app services.

**Reply by awalias** [reply]
>where the Canada datacenter is located?

Montreal

**Reply by ShravanSunder** [reply]
@awalias if we call supabase postgres from a aws resource in the same region, is it considered data egress?
is there a definition of egress for supabase?  is it the same as aws's definiiton?

**Comment by yangcheng** [comment]
are there plans to add support for us-west-2?  It's relative easier to get GPU and other larger instance in us-west-2 on AWS. I'd like to keep the database closer
